integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py

```diff
@@ -287,6 +288,40 @@ def handle(self, span: LangfuseSpan, component_type: Optional[str]) -> None:
             coerced_output = tracing_utils.coerce_tag_value(span.get_data().get(_PIPELINE_OUTPUT_KEY))
             span.raw_span().update(input=coerced_input, output=coerced_output)
 
+        # special case for ToolInvoker (to update the span name to be: `original_component_name - [tool_names]`)
+        if component_type == "ToolInvoker":
+            tool_input_data = span.get_data().get("haystack.component.input")
```

Comment:
To be consistent with the rest of the code base let's store `"haystack.component.input"` in a separate variable called `_COMPONENT_INPUT_KEY`